cfgov/regulations3k/models/pages.py

```diff
@@ -257,7 +258,14 @@ def get_context(self, request, *args, **kwargs):
         )
         context.update({
             'regulation': self.regulation,
-            'breadcrumb_items': self.get_breadcrumbs(request)
+            'breadcrumb_items': self.get_breadcrumbs(request),
```

Comment:
I don't think this is exactly the right place for this comment, but it's breadcrumb-ish, so here goes!  When I'm viewing a section page of a previous version of a regulation, the breadcrumb that points back to the regulation landing page takes me to the current-law version of the landing page instead of the previous-version landing page. I'm not sure this is totally crucial to fix right now, though.

Reply:
Hmm, yeah, I see that. I'll see what I can do.

Reply:
Turns out, yup, fixable!